src/citrine/resources/data_concepts.py

```diff
@@ -359,15 +146,25 @@ def _make_class_dict():
         from citrine.resources.material_run import MaterialRun
         from citrine.resources.measurement_run import MeasurementRun
         from citrine.resources.process_run import ProcessRun
-        _clazz_list = [Condition, Parameter, Property, PropertyAndConditions,
-                       ConditionTemplate, ParameterTemplate, PropertyTemplate,
+        _clazz_list = [ConditionTemplate, ParameterTemplate, PropertyTemplate,
                        MaterialTemplate, MeasurementTemplate, ProcessTemplate,
                        IngredientSpec, MaterialSpec, MeasurementSpec, ProcessSpec,
                        IngredientRun, MaterialRun, MeasurementRun, ProcessRun]
         for clazz in _clazz_list:
             DataConcepts.class_dict[clazz._response_key] = clazz
         DataConcepts.class_dict['link_by_uid'] = LinkByUID
 
+    @classmethod
+    def get_encoder(cls):
+        """Get a taurus-compatible json serializer/deserializer."""
+        if cls.encoder is None:
+            DataConcepts._make_class_dict()
+            cls.encoder = TaurusJson()
+            cls.encoder.register_classes(
+                {k: v for k, v in DataConcepts.class_dict.items() if k != "link_by_uid"}
```

Comment:
This is where we override the taurus objects with their citrine-python equivalents